### PR TITLE
Fixes #16726: Add a remove command to rudder-synchronize

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/16726

### CODE REVIEW

**Comment by Fdall** [comment]
PR updated with a new commit

File: scripts/rudder-synchronize [hunk 1]
```diff
@@ -10,6 +10,7 @@ Only works on Rudder 6.0+
 Usage:
     rudder-synchronize export <rudder_item> <uuid> <path> [--debug]
     rudder-synchronize import <rudder_item> <path> [--debug]
+    rudder-synchronize delete <rudder_item> <uuid> [--debug]
```
Comment (amousset) [inline scripts/rudder-synchronize]: A non-interactive option would be good

Comment (peckpeck) [reply scripts/rudder-synchronize]: indeed

**Comment by Fdall** [comment]
PR updated with a new commit

**Pull request merged**